go.mod

```diff
@@ -1,5 +1,12 @@
 module github.com/ava-labs/hypersdk
 
+// - Changes to the minimum golang version must also be replicated in:
+//   - CONTRIBUTING.md
+//   - examples/morpheusvm/go.mod
+//   - go.mod (here)
```

Comment:
Quick search through the repo, there are a few further places that we should mention here:

- Dockerfile
- devcontainer.json (only for minor version bumps)